Streams.subproj/MPWURLFetchStream.h

```diff
@@ -23,6 +23,7 @@
 @property (nonatomic, readonly) NSMutableSet *inflight;
 @property (nonatomic, readonly) NSURLSession *downloader;
 @property (assign)  int maxInflight;
+@property (nonatomic, strong) NSThread *targetThread;
```

Comment:
<img width="650" alt="image" src="https://user-images.githubusercontent.com/17592342/31962859-0d5f1b22-b8ff-11e7-8446-4d4c21c4f203.png">

I think there might be a reference leak here. Can you plz confirm?
Thanks

Reply:
i thought a bit more about this, i think we are "probably" ok, but would like your confirmation.